### PR TITLE
Fix integration tests

### PR DESCRIPTION
**Pull request by cuixq**
Update the expected responses in API integration tests:
 - add a new Go vulnerability to test response
 - versions that we test against are no longer affected

### CODE REVIEW

File: gcp/api/integration_tests.py [hunk 1]
```diff
@@ -485,12 +487,7 @@ def test_query_purl(self):
 
     self.assert_results_equal({'vulns': another_expected}, response.json())
 
-    expected_deb = [
-        self._get('CVE-2009-4487'),
-        self._get('CVE-2013-0337'),
-        self._get('DLA-3203-1'),
-        self._get('DSA-4921-1')
-    ]
+    expected_deb = [self._get('DLA-3203-1'), self._get('DSA-4921-1')]
```
Comment (cuixq) [inline gcp/api/integration_tests.py:490]: @andrewpollock I noticed you had a PR https://github.com/google/osv.dev/pull/2278 that added the two vulns that are going to be removed in this PR - do you know if this is expected?

Comment (andrewpollock) [reply gcp/api/integration_tests.py:490]: These CVEs started being introduced from the additional Debian conversion by @hogo6002 
If they're not converting any more we should figure out why that's the case

Comment (hogo6002) [reply gcp/api/integration_tests.py:490]: The CVE (https://osv.dev/vulnerability/CVE-2009-4487) entries are still on OSV.dev, but they no longer list this queried version(`1.14.2-2+deb10u3`) as an affected version.

Comment (cuixq) [reply gcp/api/integration_tests.py:490]: so is this something we need to worry about? or this is intended?

**Pull request merged**